definitions/npm/react-helmet_v6.x.x/test_react-helmet.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Helmet from 'react-helmet';
+const HelmetComponent = () => (
+  <Helmet
+    htmlAttributes={{"lang": "en", "amp": undefined}} // amp takes no value
+    title="My Title"
+    titleTemplate="MySite.com - %s"
+    defaultTitle="My Default Title"
+    base={{"target": "_blank", "href": "http://mysite.com/"}}
+    meta={[
+      {"name": "description", "content": "Helmet application"},
+      {"property": "og:type", "content": "article"}
+    ]}
+    link={[
+      {"rel": "canonical", "href": "http://mysite.com/example"},
+      {"rel": "apple-touch-icon", "href": "http://mysite.com/img/apple-touch-icon-57x57.png"},
+      {"rel": "apple-touch-icon", "sizes": "72x72", "href": "http://mysite.com/img/apple-touch-icon-72x72.png"}
+    ]}
+    script={[
+      {"src": "http://include.com/pathtojs.js", "type": "text/javascript"},
+      {"type": "application/ld+json", innerHTML: `{ "@context": "http://schema.org" }`}
+    ]}
+    onChangeClientState={(newState) => console.log(newState)}
+  />
+)
+
+
+const fail = () => (
```

Comment:
Preferred way of tests it to break down into describe/it blocks. eg: https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/koa-basic-auth_v4.x.x/flow_v0.83.x-/test_koa-basic-auth_v4.x.x.js

Reply:
@Brianzchen See above :smile:  What do we do? Keep? Remove?

Reply:
Personally I think it's fine because you copied from an older version. But here are some docs referring test standards with new lib defs https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope

Reply:
Suppression codes added to v3 and v5!